samples/tasks/scenarios/ovn-network/osh_workload_incremental.json

```diff
@@ -160,6 +160,62 @@
                 }
             ]
         },
+        {
```

Comment:
This can be simplified, quite a few of the parameters are not used by the `setup_switch_per_node_init` workload.

Also, as we don't really care that much about measuring how much each of the container creation/ext ns iterations take we can just execute all of them in a batch.

I'd suggest the following incremental change:
```
diff --git a/samples/tasks/scenarios/ovn-network/osh_workload_incremental.json b/samples/tasks/scenarios/ovn-network/osh_workload_incremental.json
index d72e289..f201c66 100644
--- a/samples/tasks/scenarios/ovn-network/osh_workload_incremental.json
+++ b/samples/tasks/scenarios/ovn-network/osh_workload_incremental.json
@@ -175,20 +175,16 @@
                             "ovn_monitor_all": {{ovn_monitor_all}},
                             "ovn_cluster_db": {{ovn_cluster_db}},
                             "max_timeout_s": 10,
-                            "batch_size": {{node_batch_size}}
+                            "batch_size": {{farm_nodes}}
                         },
                         "lnetwork_create_args": {
-                            "networks_per_router": {{networks}},
-                            "start_gw_cidr": "2.0.0.0/16",
                             "start_ext_cidr": "3.0.0.0/16",
                             "gw_router_per_network": {{gw_router}},
-                            "batch": {{gw_batch_ops}},
-                            "cluster_cidr": "16.0.0.0/4"
                         }
                     },
                     "runner": {
                         "type": "serial",
-                        "times": {{farm_nodes // node_batch_size}}
+                        "times": 1
                     },
                     "sla": {
                         "max_seconds_per_iteration": {{sla}}
@@ -199,19 +195,6 @@
                         },
                         "ovn_nb": {},
                         "sandbox": { "tag": "ToR1" },
-                        "datapath": {
-                            "router_create_args": {
-                                "amount": 1
-                            }
-                        },
-                        "ovn-nbctld" : {
-                            "daemon_mode": True,
-                            "remote": {{ovn_central_ip}},
-                            "prot": "ssl",
-                            "privkey": "/opt/ovn/ovn-privkey.pem",
-                            "cert": "/opt/ovn/ovn-cert.pem",
-                            "cacert": "/opt/ovn/pki/switchca/cacert.pem"
-                        }
                     }
                 }
             ]
```